File: LecturesAttendanceSystem/LecturesAttendanceSystem.Services/Dtos/WorkWeekDTO.cs
using System;
using LecturesAttendanceSystem.Data.Entities;

namespace LecturesAttendanceSystem.Services.Dtos
{
    public class WorkWeekDTO
    {
        public DateTime LessonTime { get; set; }

        public CompactLessonDTO MondayLesson { get; set; }
        
        public CompactLessonDTO TuesdayLesson { get; set; }
        
        public CompactLessonDTO WednesdayLesson { get; set; }
        
        public CompactLessonDTO ThursdayLesson { get; set; }
        
        public CompactLessonDTO FridayLesson { get; set; }
    }
}

File: lecturesattendancesystem-ui/src/app/models/workWeekDTO.ts
import {CompactLessonDTO} from "./compactLessonDTO";

export interface WorkWeekDTO {
  lessonTime: Date,
  mondayLesson: CompactLessonDTO,
  tuesdayLesson: CompactLessonDTO,
  wednesdayLesson: CompactLessonDTO,
  thursdayLesson: CompactLessonDTO,
  fridayLesson: CompactLessonDTO
}
